src/core/cigar.rs

```diff
@@ -4,10 +4,10 @@ use crate::error::{err, Error, Result};
 
 #[derive(Debug, Clone, PartialEq)]
 pub struct Cigar {
-    raw: Vec<u8>,
-    code: String,
-    size: u32,
-    verfer: Verfer,
+    pub(crate) raw: Vec<u8>,
+    pub(crate) code: String,
+    pub(crate) size: u32,
+    pub(crate) verfer: Verfer,
```

Comment:
I guess this is still required for uniffi? Better than pub.

Reply:
Yes. For `uniffi`....